src/tests/ftest/util/soak_utils.py

```diff
@@ -66,11 +66,13 @@ def add_pools(self, pool_names, ranks=None):
     target_list = ranks if ranks else None
     for pool_name in pool_names:
         path = "".join(["/run/", pool_name, "/*"])
-        properties = self.params.get('properties', path, "")
+        properties = self.params.get('properties', path, None)
         # allow yaml pool property to override the scrub default; whether scrubber is enabled or not
-        if self.enable_scrubber and "scrub" not in properties:
+        if self.enable_scrubber and "scrub" not in str(properties):
             scrubber_properties = "scrub:timed,scrub_freq:120"
-        params['properties'] = (",").join(filter(None, [properties, scrubber_properties]))
+            params['properties'] = (",").join(filter(None, [properties, scrubber_properties]))
```

Comment:
No need for parens here
```suggestion
            params['properties'] = ",".join(filter(None, [properties, scrubber_properties]))
```